schema_based_on_diagram.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE patients (id INT PRIMARY KEY, name VARCHAR(70), date_of_birth DATE);
+CREATE TABLE medical_histories (id INT PRIMARY KEY, admitted_at TIMESTAMP, patient_id INT, status VARCHAR(50), FOREIGN KEY (patient_id) REFERENCES patients(id));
+CREATE TABLE treatments (id INT PRIMARY KEY, type VARCHAR(50),name VARCHAR(50));
```

Comment:
- [ ] Kindly note that there is a many-to-many relationship between the `medical_histories` and `treatments` tables. Therefore it would be nice to have a **JOIN TABLE** `medical_histories_treatments` reflecting the **many-to-many** relationship with `medical_history_id` and `treatment_id` as foreign keys referencing the `medical_histories` and `treatments` tables respectively